### PR TITLE
Hello Microverse Project: Add homepage

### PR DESCRIPTION
**Pull request by Graycemuthui**
- Created a repository on GitHub.
-  Added linter configurations.
- Created .gitignore file and added test.md file.
- Created HTML file with a heading.
- Created a CSS file and added style to the heading.
- Used proper GitHub flow.
- Followed all project and general guidelines.

Kindly review and provide feedback.
Thanks.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+
+<body>
+    <h1>Hello Microverse</h1>
+</body>
```
Comment (mahbubul14) [inline index.html:14]: - [ ] For readability, please add two spaces of indentation. We should not use the tab key. You can check [HTML & CSS best practices. ](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md)

**Review by mahbubul14 — CHANGES_REQUESTED** [review]
Hi @Graycemuthui ,

Good job so far! :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!

# Required Changes ♻️

- [ ] The `README` file should be descriptive. Please check [this template ](https://github.com/microverseinc/readme-template) and update your `README` file.

###  To Highlights
:heavy_check_mark: No linters error.
:heavy_check_mark: Correct github-flow.
:heavy_check_mark: Descriptive PR title & description.

### Optional suggestions

- [ ] Check the comments under the review for more suggestions. :arrow_down: 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. You can also reach me on [slack](https://microverse-students.slack.com/team/U01J39GL614)

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by roshan-bajgain — APPROVED** [review]
# Approved 👏 

Hi @Graycemuthui , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

To Highlight: 
- The Pull Request has a proper title and description. ✔️ 
- You have added a descriptive Readme file. Good Job. ✔️ 
- There is no linter error at present. ✔️ 
- Project has met all the technical aspects you needed for this project. Keep rocking! ✔️ 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of code reviews per project which you can see from your dashboard. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)_

**Comment by Graycemuthui** [comment]
@mahbubul14 and @roshan-bajgain 
Thank you so much for the reviews.

**Pull request merged**